website/src/data/users.tsx

```diff
@@ -383,6 +383,14 @@ const Users: User[] = [
     source: 'https://github.com/loft-sh/devspace',
     tags: ['opensource'],
   },
+  {
+    title: 'Djamaile Rahamat',
+    description: 'Djamaile Rahamat Blog on making cool stuff',
+    preview: require('./showcase/djamaile.png'),
+    website: 'https://djamail.dev/',
```

Comment:
This url does not seem to be online.
We only add to showcase sites that are online :)

Reply:
It is online, was a silly spelling mistake. My bad!